website/docs/reference/snowflake-permissions.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Snowflake Permissions"
+---
+
+## dbt Labs recommended Snowflake permissions
+
+```
+-- NOTE: warehouse_name, database_name, and role_name are placeholders! Replace as-needed for your organization's naming convention!
```

Comment:
Suggesting a newline in the comment since there isn't line wrapping in the browser. See screenshot below for what it looks like currently without this suggestion: 
```suggestion
-- NOTE: warehouse_name, database_name, and role_name are placeholders!
-- Replace as-needed for your organization's naming convention!
```

<img width="847" alt="image" src="https://user-images.githubusercontent.com/44704949/189750197-22cd5559-8081-4cb3-ba8d-69dbc8c35328.png">